common/persistence/tests/mysql_test_util.go

```diff
@@ -56,8 +56,8 @@ const (
 
 	// TODO hard code this dir for now
 	//  need to merge persistence test config / initialization in one place
```

Comment:
Does this mean we're no longer testing v57?

Reply:
In reality, we don't test against MySQL 5.7 anywhere. All docker container is using MySQL 8. What it means is that some tests won't run with the old visibility interface that supports MySQL v5.7.

Reply:
If we aren't testing MySQL 5.7 anywhere, but we provide a library for it, that's risky for our users. Fixing that is outside the scope of this PR, but do we have a follow-up task, like sunsetting 5.7 support altogether?

Reply:
For v1.21, removing support for MySQL 5.7.